From edd0be1655dee99dc8a7aa9ce0d040f6b371e095 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 11 Sep 2023 23:36:03 +0700
Subject: [PATCH] configs: rzf_defconfig: enable ECC Detection and Correction
 mode support for RZ/Five

Enable ECC single bit error correction multibit detection mode support
for RZ/Five SMARC and Development EVK DDR4 Boards.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 configs/rzf-dev_ddr4_defconfig | 1 +
 configs/smarc-rzf_defconfig    | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/rzf-dev_ddr4_defconfig b/configs/rzf-dev_ddr4_defconfig
index 2145f9c777..8c1233bdae 100644
--- a/configs/rzf-dev_ddr4_defconfig
+++ b/configs/rzf-dev_ddr4_defconfig
@@ -145,3 +145,4 @@ CONFIG_SBI_V02=y
 CONFIG_CMD_EXCEPTION=y
 CONFIG_CMD_RZG2L_ECC=y
 CONFIG_RZF_DDR_ECC=y
+CONFIG_RZF_DDR_ECC_DETECT_CORRECT=y
diff --git a/configs/smarc-rzf_defconfig b/configs/smarc-rzf_defconfig
index 102fd9ce29..f5734e8a00 100644
--- a/configs/smarc-rzf_defconfig
+++ b/configs/smarc-rzf_defconfig
@@ -156,3 +156,4 @@ CONFIG_SPI_FLASH_STMICRO=y
 CONFIG_DM_SPI_FLASH=y
 CONFIG_CMD_RZG2L_ECC=y
 CONFIG_RZF_DDR_ECC=y
+CONFIG_RZF_DDR_ECC_DETECT_CORRECT=y
-- 
2.25.1

